src/app/containers/MediaPageMain/__snapshots__/index.test.jsx.snap

```diff
@@ -0,0 +1,886 @@
+  width: 100%;
+  max-width: 10.496849999999998rem;
+  min-width: 6.9979rem;
+}
+
+.c4 {
+  box-sizing: content-box;
+  color: #FFFFFF;
+  fill: currentColor;
+  padding-top: 1rem;
+  padding-bottom: 0.75rem;
+  height: 1.5rem;
+  width: 100%;
+  max-width: 10.496849999999998rem;
+  min-width: 6.9979rem;
+}
+
+.c2:hover .c4,
+.c2:focus .c4 {
+  -webkit-text-decoration: none;
+  text-decoration: none;
+  border-bottom: 0.25rem solid #FFFFFF;
+}
+
+.c11 {
+  padding: 0.5rem 0 0.5rem;
+  color: #FFFFFF;
+  font-weight: 700;
+  -webkit-text-decoration: none;
+  text-decoration: none;
+  display: block;
+}
+
+.c14 {
+  padding: 0.5rem 0 0.5rem;
+  color: #FFFFFF;
+  font-weight: 700;
+  -webkit-text-decoration: none;
+  text-decoration: none;
+  display: inline;
+}
+
+.c10:hover .c12,
+.c10:focus .c12 {
+  padding-bottom: 2px;
+  border-bottom: 2px solid #FFFFFF;
+}
+
+.c8 {
+  border-bottom: 1px solid #3F3F42;
+  display: grid;
+  grid-auto-flow: column;
+  list-style-type: none;
+  margin: 0;
+  padding: 0 0 0.5rem;
+}
+
+.c8 > li:first-child {
+  border-bottom: 1px solid #3F3F42;
+  padding: 0.5rem 0;
+  margin-bottom: 0.5rem;
+  grid-column: 1/-1;
+}
+
+.c9 {
+  min-width: 50%;
+}
+
+.c6 {
+  background-color: #222222;
+  font-size: 0.875rem;
+  line-height: 1rem;
+}
+
+.c7 {
+  max-width: 80rem;
+  margin: 0 auto;
+}
+
+.c13 {
+  color: #FFFFFF;
+  margin: 0;
+  padding: 1rem 0;
+}
+
+@media (min-width:25rem) {
+  .c0 {
+    height: 5rem;
+    padding: 0 1rem;
+  }
+}
+
+@media (min-width:25rem) {
+  .c4 {
+    padding-top: 1.75rem;
+    padding-bottom: 1.5rem;
+  }
+}
+
+@media screen and (-ms-high-contrast:active),print {
+  .c4 {
+    fill: windowText;
+  }
+}
+
+@media (max-width:14.9375rem) {
+  .c8 {
+    grid-auto-flow: row;
+  }
+}
+
+@media (min-width:15rem) and (max-width:37.4375rem) {
+  .c8 {
+    grid-column-gap: 0.5rem;
+    grid-template-columns: repeat(2,1fr);
+    grid-template-rows: repeat( 4,auto );
+  }
+}
+
+@media (min-width:37.5rem) and (max-width:62.9375rem) {
+  .c8 {
+    grid-column-gap: 1rem;
+    grid-template-columns: repeat(3,1fr);
+    grid-template-rows: repeat( 3,auto );
+  }
+}
+
+@media (min-width:63rem) and (max-width:80rem) {
+  .c8 {
+    grid-column-gap: 1rem;
+    grid-template-columns: repeat(4,1fr);
+    grid-template-rows: repeat( 3,auto );
+  }
+}
+
+@media (min-width:80rem) {
+  .c8 {
+    grid-column-gap: 1rem;
+    grid-template-columns: repeat(5,1fr);
+    grid-template-rows: repeat( 3,auto );
+  }
+}
+
+@supports not (display:grid) {
+  .c8 > li:first-child {
+    width: 100%;
+  }
+}
+
+@supports not (display:grid) {
+  .c9 {
+    display: inline-block;
+  }
+}
+
+@media (max-width:25rem) {
+  .c6 {
+    padding: 0 0.5rem;
+  }
+}
+
+@media (min-width:25rem) {
+  .c6 {
+    padding: 0 1rem;
+  }
+}
+
+@media (min-width:20rem) and (max-width:37.4375rem) {
+  .c6 {
+    line-height: 1.125rem;
+  }
+}
+
+@media (min-width:37.5rem) {
+  .c6 {
+    font-size: 0.8125rem;
+  }
+}
+
+<footer
```

Comment:
I'm wondering why the `MediaPageMain` snapshot has the footer in it? 🤔 Should just be the content inside of `<main>`